tools/text_processing/text_processing/awk.xml

```diff
@@ -1,48 +1,126 @@
-<tool id="tp_awk_tool" name="Text reformatting" version="@BASE_VERSION@.1">
+<tool id="tp_awk_tool" name="Text reformatting" version="@BASE_VERSION@.2">
     <description>with awk</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <requirements>
-        <requirement type="package" version="4.2.0">gawk</requirement>
+        <requirement type="package" version="5.0.1">gawk</requirement>
     </requirements>
     <version_command>awk --version | head -n 1</version_command>
     <command>
 <![CDATA[
+        #set $tool_input = 0
         env -i
+        #for env in $envs
```

Comment:
why not set this variables with the `-v` parameter of awk?

Would `&&` or `;` after `env -i` be good?

Reply:
Ah, OK.